pycbc/inference/models/relbin.py

```diff
@@ -526,20 +519,22 @@ def _loglr(self):
             # project waveform to detector frame if waveform does not deal
             # with detector response. Otherwise, skip detector response.
 
-            if self.no_det_response:
-                dtc = -end_time
+            if self.apply_det_response:
                 channel = wfs[ifo].numpy()
-                filter_i, norm_i = lik(freqs, dtc, channel, h00,
+                filter_i, norm_i = lik(freqs, 0.0, channel, h00,
                                        sdat['a0'], sdat['a1'],
                                        sdat['b0'], sdat['b1'])
             else:
                 hp, hc = wfs[ifo]
                 det = self.det[ifo]
                 fp, fc = det.antenna_pattern(p["ra"], p["dec"],
-                                             p["polarization"], times)
+                                             0.0, times)
                 dt = det.time_delay_from_earth_center(p["ra"], p["dec"], times)
-                dtc = p["tc"] + dt - end_time
+                dtc = p["tc"] + dt - end_time - self.ta[ifo]
 
+                f = (fp + 1.0j * fc) * pol_phase
```

Comment:
Is it faster this way? It seems that you still have the call to antenna_pattern ... Otherwise I don't understand the pol_phase change?